src/sagemaker_inference/transformer.py

```diff
@@ -19,6 +19,11 @@
 import importlib
 import traceback
 
+from sagemaker_inference import logging
+
+logging.configure_logger()
+logger = logging.get_logger()
```

Comment:
Nit: is there a specific reason to use logger like this - it seems this is the only file using `logging` from sagemaker_inference. If simple logging works, would recommend using that: https://github.com/aws/sagemaker-pytorch-inference-toolkit/blob/master/src/sagemaker_pytorch_serving_container/ts_environment.py#L24.

Reply:
Makes sense, will update